net-proxy/torsocks/torsocks-1.3.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A transparent socks proxy for use with tor"
+DESCRIPTION="Torsocks allows you to use most socks-friendly applications \
+in a safe way with Tor. It ensures that DNS requests are handled safely \
+and explicitly rejects UDP traffic from the application you're using."
+HOMEPAGE="https://www.torproject.org"
+COPYRIGHT="2000-2008 Shaun Clowes
+	2008-2011 Robert Hogan
+	2005 Total Information Security Ltd.
+	2010 Alex Rosenberg
+	2004, 2006 Peter Palfrader
+	2000 Alessandro Iurlano
+	2004 Tomasz Kojm"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/dgoulet/torsocks/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="09debf9e35d70802fdb0e56335bacab65e84fad615f68d7006539713c1698123"
+PATCHES="torsocks-$portVersion.patchset"
+GLOBAL_WRITABLE_FILES="settings/torsocks.conf manual"
+
+ARCHITECTURES="x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	torsocks$secondaryArchSuffix = $portVersion
+	cmd:torsocks$secondaryArchSuffix
+	cmd:usewithtor$secondaryArchSuffix
+	lib:libtorsocks$secondaryArchSuffix = 1.0.0 compat >= 0
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix >= $haikuVersion
```

Comment:
You can (or should?) drop the «**` >= $haikuVersion`**» for «**`haiku$secondaryArchSuffix`**» and «**`haiku${secondaryArchSuffix}_devel`**».